ARTCC/Danger Areas/EG D148.txt

```diff
@@ -0,0 +1,9 @@
+;EGD148
+
+;EGD148     N051.17.44.000 W002.10.16.000 N051.19.37.000 W002.07.45.000
+;EGD148     N051.19.37.000 W002.07.45.000 N051.20.55.000 W002.04.10.000
+;EGD148     N051.20.55.000 W002.04.10.000 N051.19.46.000 W002.03.05.000
+;EGD148     N051.19.46.000 W002.03.05.000 N051.17.05.000 W002.03.12.000
+;EGD148     N051.17.05.000 W002.03.12.000 N051.16.02.000 W002.06.57.000
+;EGD148     N051.16.02.000 W002.06.57.000 N051.17.01.000 W002.09.29.000
+;EGD148     N051.17.01.000 W002.09.29.000 N051.17.44.000 W002.10.16.000
```

Comment:
For these lines: remove the ;

Otherwise the lines will be seen by ES as commented out and it won't display anything :)

Reply:
ah ok, will sort :)